specification/datalake-store/data-plane/readme.typescript.md

```diff
@@ -0,0 +1,13 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+``` yaml $(typescript)
+typescript:
+  azure-arm: true
+  package-name: "@azure/arm-datalake-store"
+  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-datalake-store"
+  source-code-folder-path: lib/filesystem
```

Comment:
We talked about not adding this file, but I'm okay with adding this file. Please manually compare this file to the readme.nodejs.md version to make sure that the properties are correct.